pkg/kubeapiserver/options/authentication.go

```diff
@@ -217,9 +217,8 @@ func (o *BuiltInAuthenticationOptions) Validate() []error {
 	}
 
 	if o.WebHook != nil {
-		retryBackoff := o.WebHook.RetryBackoff
-		if retryBackoff != nil && retryBackoff.Steps <= 0 {
-			allErrors = append(allErrors, fmt.Errorf("number of webhook retry attempts must be greater than 1, but is: %d", retryBackoff.Steps))
+		if o.WebHook.RetryBackoff != nil && o.WebHook.RetryBackoff.Steps <= 0 {
+			allErrors = append(allErrors, fmt.Errorf("number of webhook retry attempts must be greater than 0, but is: %d", o.WebHook.RetryBackoff.Steps))
```

Comment:
I think the original code is better. The chain of referring is hard to read.

Reply:
ok, fixed.